hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirEncryptionZoneOp.java

```diff
@@ -537,12 +537,12 @@ static boolean isInAnEZ(final FSDirectory fsd, final INodesInPath iip)
    * then launch up a separate thread to warm them up.
    */
   static void warmUpEdekCache(final ExecutorService executor,
-      final FSDirectory fsd, final int delay, final int interval) {
+      final FSDirectory fsd, final int delay, final int interval, final int maxRetries) {
```

Comment:
Can you edit a comment in the function documentation to indicate that the warm up is best effort.

Reply:
add the javadoc